src/clients/Transmission.ts

```diff
@@ -186,13 +189,18 @@ export default class Transmission implements TorrentClient {
 
 		let addResponse: TorrentAddResponse;
 
+		const skipRecheckTorrent =
+			decision === Decision.MATCH_PARTIAL
+				? skipRecheck
+				: true;
```

Comment:
what are you going for with these changes? It looks like you're trying to preserve original behavior, but I think the real story of the original behavior is just that we never finished implementing skipRecheck.

Reply:
Yes I wanted to keep original behavior but add torrent in paused state when using match partial, unless they enable skipRecheck. The only place this was necessary was for .torrent matches for partial as data matches already handled this.